src/sqlitestorage.cpp

```diff
@@ -1067,14 +1068,43 @@ int SqliteStorage::loadContactIncidences(const Person &person, int limit, QDateT
     }
     SL3_bind_int64(stmt1, index, secsStart);
 
-    count = d->loadIncidences(stmt1, true, limit, last, false);
+    count = d->loadIncidences(stmt1, true, nullptr, limit, last, false);
 
 error:
     d->mIsLoading = false;
 
     return count;
 }
 
+bool SqliteStorage::search(const QString &key, QStringList *identifiers)
+{
+    if (!d->mDatabase || key.isEmpty())
+        return false;
+
+    d->mIsLoading = true;
+    const char *query1 = SEARCH_COMPONENTS;
+    int qsize1 = sizeof(SEARCH_COMPONENTS);
+    const QByteArray s('%' + key.toUtf8() + '%');
```

Comment:
Literal "%" search at least could need some escaping.

Reply:
I was misleading when adding in test a search with `'` and `%` inside but of course `%` worked since it matches everything (I didn't create the test well enough to check that it didn't return too many match). Arf, you're right, I need escaping here. Thanks.